python_modules/dagster/dagster/core/telemetry.py

```diff
@@ -56,14 +60,26 @@
 }
 
 
-def telemetry_wrapper(f):
+def telemetry_wrapper(metadata):
```

Comment:
@smackesey did a nice trick in #5943 for writing decorators that take optional args if we stick with this path

Reply:
More specifically here: https://github.com/dagster-io/dagster/pull/5943#discussion_r780426455

Or see `whitelist_for_serdes` definition on master.